qutebrowser/browser/qutescheme.py

```diff
@@ -201,6 +201,26 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+def _tab_fields_to_tabs_page_info(fields):
+    return (fields['title'], fields['current_url'])
+
+@add_handler('tabs')
+def qute_tab_list(_url):
+    """Handler for qute://tabs. Display information about all open tabs."""
+    tabs = {}
+    for win_id in objreg.window_registry:
+        win_id_str = str(win_id)
+        tabs[win_id_str] = []
+        tabbed_browser = objreg.get('tabbed-browser', scope='window', window=win_id)
```

Comment:
A couple of these lines are too long, and triggering pylint failures, could you wrap them please?

https://travis-ci.org/qutebrowser/qutebrowser/jobs/323561046